### PR TITLE
Fix GCP resource reference information

### PR DESCRIPTION
**Pull request by jwitz**
Based on https://astronomer.slack.com/archives/C01UA03TPV1/p1661809539941169

I took a guess as to which of the two supported GCP nodes we use as the default, definitely need a GCP SME to check me on this one!

### CODE REVIEW

File: astro/resource-reference-gcp.md [hunk 1]
```diff
@@ -22,9 +22,9 @@ Read the following document for a reference of our default resources as well as
...
-| Astro node pool | A node pool runs all proprietary Astronomer components. This node pool is fully managed by Astronomer| 1x pool of m5.xlarge nodes |
-| Airflow node pool | An node pool runs all core Airflow components such as the scheduler and webserver. This node pool is fully managed by Astronomer | 1x pool of m5.xlarge nodes |
+| Worker node pools | Node pools run all Airflow workers. The number of nodes in the pool auto-scales based on the demand for workers in your cluster. You can configure multiple worker node pools to run tasks on different instance types| 1x pool of e2-standard-4 nodes |
+| Astro node pool | A node pool runs all proprietary Astronomer components. This node pool is fully managed by Astronomer| 1x pool of e2-standard-4 nodes |
+| Airflow node pool | A node pool runs all core Airflow components such as the scheduler and webserver. This node pool is fully managed by Astronomer | 1x pool of e2-standard-4 nodes |
```
Comment (jwitz) [inline astro/resource-reference-gcp.md:27]:
```suggestion
| Worker node pools | Node pools run all Airflow workers. The number of nodes in the pool auto-scales based on the demand for workers in your cluster. You can configure multiple worker node pools to run tasks on different instance types| 1x pool of e2-standard-4 nodes |
| Astro node pool | A node pool runs all proprietary Astronomer components. This node pool is fully managed by Astronomer| 1x pool of n2-standard-4 nodes |
| Airflow node pool | A node pool runs all core Airflow components such as the scheduler and webserver. This node pool is fully managed by Astronomer | 1x pool of n2-standard-4 nodes |
```

File: astro/resource-reference-gcp.md [hunk 2]
```diff
@@ -22,9 +22,9 @@ Read the following document for a reference of our default resources as well as
...
-| Astro node pool | A node pool runs all proprietary Astronomer components. This node pool is fully managed by Astronomer| 1x pool of m5.xlarge nodes |
-| Airflow node pool | An node pool runs all core Airflow components such as the scheduler and webserver. This node pool is fully managed by Astronomer | 1x pool of m5.xlarge nodes |
+| Worker node pools | Node pools run all Airflow workers. The number of nodes in the pool auto-scales based on the demand for workers in your cluster. You can configure multiple worker node pools to run tasks on different instance types| 1x pool of e2-standard-4 nodes |
+| Astro node pool | A node pool runs all proprietary Astronomer components. This node pool is fully managed by Astronomer| 1x pool of n2-standard-4 nodes |
+| Airflow node pool | A node pool runs all core Airflow components such as the scheduler and webserver. This node pool is fully managed by Astronomer | 1x pool of n2-standard-4 nodes |
```
Comment (jwitz) [inline astro/resource-reference-gcp.md:27]:
```suggestion
| Astro node pool | A node pool runs all proprietary Astronomer components. This node pool is fully managed by Astronomer| 1x pool of e2-standard-4 nodes |
| Airflow node pool | A node pool runs all core Airflow components such as the scheduler and webserver. This node pool is fully managed by Astronomer | 1x pool of e2-standard-4 nodes |
```

File: astro/install-aws.md [hunk 3]
```diff
@@ -195,7 +195,7 @@ After creating the AWS account, provide Astronomer support with the following in
 - Your preferred node instance type.
 - Your preferred maximum node count.
 
-If you do not specify configuration preferences, Astronomer will create a cluster with two `m5.xlarge` nodes and a maximum node count of 20 in `us-east-1`. For information on all supported regions, configurations, and defaults, see [Resources required for Astro on AWS](resource-reference-aws.md).
+If you do not specify configuration preferences, Astronomer will create a cluster with `m5.xlarge` nodes and a maximum node count of 20 in `us-east-1`. For information on all supported regions, configurations, and defaults, see [Resources required for Astro on AWS](resource-reference-aws.md).
```
Comment (kwcanuck) [inline astro/install-aws.md:198]:
```suggestion
If you do not specify configuration preferences, Astronomer creates a cluster with `m5.xlarge` nodes and a maximum node count of 20 in `us-east-1`. For information on all supported regions, configurations, and defaults, see [Resources required for Astro on AWS](resource-reference-aws.md).
```

**Review by kwcanuck — APPROVED** [review]

File: astro/resource-reference-azure.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 sidebar_label: "Azure"
 title: "Resources required for Astro on Azure"
 id: resource-reference-azure
-description: Reference information for new Astro on Azure clusters.
+description: Reference of all supported configurations for new clusters on Astro in Azure.
```
[the remark this answers is not in the document]
Comment (jwitz) [reply astro/resource-reference-azure.md:5]: I wanted to make this consistent with our other descriptions, as they all appear together on the same index menu

**Pull request merged**